Addition of new auth-method fields to consul cli commands section (#2769)
hashicorp/web-unified-docs: content/consul/v2.0.x/content/commands/acl/auth-method/create.mdx

  should produce. This can be either 'local' or 'global'. If empty the value of
  'local' is assumed. Added in Consul 1.8.0.

- `-token-name-format=<string>` - Defines the token name format for the generated
   tokens This can be lightly templated using HIL ${foo} syntax. 

>    Defaults to ${auth_method_type}-${auth_method_name}.

Reviewer: Add monospace formatting. Also applies to other versions.

```suggestion
- `-token-name-format=<string>` - Defines the token name format for the generated
   tokens This can be lightly templated using HIL `${foo}` syntax. 
   Defaults to `${auth_method_type}-${auth_method_name}`.
```

  For all auth_method_type values, by default following variables are available for interpolation:
    - `auth_method_type`
    - `auth_method_name`

  Additional for OIDC and JWT claims mapping values are also available in the form of `value.<claim_mapping_value>`.
  For an OIDC auth method - `given_name` and `family_name` claim mapping are available if sent by the auth provider.
  Example OIDC TokenNameFormat
  `'${auth_method_type}-${auth_method_name}-${value.given_name}-${value.family_name}'`
  Additionally for an JWT auth method `preferred_username` and `sub` claim mapping are available if sent by the auth provider.
  Example JWT TokenNameFormat

>   `'${auth_method_type}-${auth_method_name}-${value.given_name}-${value.family_name}-${value.preferred_username}-${value.sub}'

Reviewer: This one does not render well in the preview:

Check [HERE](https://unified-docs-frontend-preview-pxep8iktv-hashicorp.vercel.app/consul/commands/acl/auth-method/create#token-name-format) and use [the API page](https://unified-docs-frontend-preview-pxep8iktv-hashicorp.vercel.app/consul/api-docs/acl/auth-methods#tokennameformat) as a reference for formatting.

### CLI

<img width="591" height="280" alt="Image" src="https://github.com/user-attachments/assets/033071d6-e3f3-40f5-a397-97f7a19da58f" />

### API

<img width="598" height="434" alt="Image" src="https://github.com/user-attachments/assets/cb536eee-0f34-461d-b50e-b4fee6508e29" />

The same applies to other versions.

Author: Changes done. 
Claim mapping feature introduced in 2.0.x.
Other versions don't have it

  should produce. This can be either 'local' or 'global'. If empty the value of
  'local' is assumed. Added in Consul 1.8.0.

- `-token-name-format=<string>` - Defines the token name format for the generated
   tokens This can be lightly templated using HIL `${foo}` syntax.

>   Defaults to `${auth_method_type}-${auth_method_name}`.

Reviewer 2: ```suggestion
- `-token-name-format=<string>` - Defines the token name format for the
   generated tokens. This can be templated using HashiCorp Interpolation
   Language. Defaults to '${auth_method_type}-${auth_method_name}'.
```

Author: Done.

  For all auth_method_type values, these variables are available for
  interpolation:
  
  - `auth_method_type`

>   - `auth_method_name`

Reviewer 2: ```suggestion
  The variables `auth_method_type` and `auth_method_name` are always available
  for interpolation.
```

Author: Done.

>   Use `value.<claim_mapping_value>` for OIDC and JWT claims mapping values.

Reviewer 2: ```suggestion
  For OIDC and JWT claims mapping values, use `value.<claim_mapping_value>`.
```

Author: Done.

  
  For an OIDC auth method, `given_name` and `family_name` claim mapping are
  available if sent by the auth provider.

  Example OIDC `TokenNameFormat`:

>   `'${auth_method_type}-${auth_method_name}-${value.given_name}-${value.family_name}'`

Reviewer 2: ```suggestion
  An OIDC auth method could provide you the `given_name` and `family_name`
  variables. Example:
  `'${auth_method_type}-${auth_method_name}-${value.given_name}-${value.family_name}'`
```

Author: Done.

  For a JWT auth method, `preferred_username` and `sub` claim
  mappings are available if sent by the auth provider. 
  
  Example JWT `TokenNameFormat`:

>   `'${auth_method_type}-${auth_method_name}-${value.given_name}-${value.family_name}-${value.preferred_username}-${value.sub}'`

Reviewer 2: ```suggestion
  А JWT auth method could provide you the `preferred_username` and `sub`
  variables. Example:

  `'${auth_method_type}-${auth_method_name}-${value.preferred_username}-${value.sub}'`
```

Author: Done.
